### DISCUSSION TITLE
FileNotFoundError - WinError 2

### DISCUSSION THREAD

**Discussion by jacoclem** [Kidel/Kathara#260] [Q&A] [answered]
Salve, eseguendo il comando kathara lstart riscontro l'errore nella foto. Ho disinstallato e reinstallato sia Docker che Kathara ma il problema persiste. Come posso risolvere ?

![Screenshot](https://github.com/KatharaFramework/Kathara/assets/128401604/c5c57feb-7710-46e9-b21c-bc9298680e72)

**Comment by tcaiazzi** [comment]
Hi @jacoclem, 

Thanks for opening the discussion. However, please write in English here so that your error can help other people.

To help you, I need some information:
1. What version of Kathara are you using?
2. Can you please attach your network scenario so that I can try to debug your error? 

Thanks, 
Tommmaso

**Reply by jacoclem** [reply]
Hi, thanks for your reply.

I'm using version 3.7.0 of kathara on windows 11, the error occurs in all my scenarios, in the attachment you can find one.

Jacopo

[lab1.zip](https://github.com/KatharaFramework/Kathara/files/13859971/lab1.zip)

**Comment by tcaiazzi** [comment]
On my Windows 11 machine, the scenario works fine, and I'm unable to reproduce your issue. So, probably your error is related to some configuration on your machine. 

To help you further, please set the logging level to `EXCEPTION`. To do so, follow these steps:
1. Open the settings: 
```bash
kathara settings
```
This command will display the following menu:
<img src="https://github.com/KatharaFramework/Kathara/assets/26346836/3ef6f01c-e255-410e-9aba-d8a4df351231" width=50% height=50%>

2. Press `6`  (and then `enter`) to select the option: ` 6 - Choose logging level to be used`.
3. Press `6`  (and then `enter`) to select the `EXCEPTION` level.
4. Press `7` (and then `enter`) to return to settings. 
5. Press `14` (and then `enter`) to exit. 

After activating the logging, please re-run the scenario and send me (in a separate file is better) the all output of the command. 

Thanks!

**Reply by jacoclem** [reply]
Hi, I followed your steps, this is the output I get when I try to run the lab. I hope I understood your request correctly.

[ouput.txt](https://github.com/KatharaFramework/Kathara/files/13864114/ouput.txt)

Thanks
Jacopo

**Reply by tcaiazzi** [reply]
It seems that there is some issue with the Powershell version you are using. 

What Powershell version are you using? 

If you don't know it, try to run the following commands and report here all the output (of both commands): 
1. `powershell.exe`
2. `pwsh.exe`

**Reply by jacoclem** [reply]
This is the output, maybe I changed something about the environment variables

[ouput.txt](https://github.com/KatharaFramework/Kathara/files/13864381/ouput.txt)

**Reply by tcaiazzi** [reply, marked as answer]
Ok, probably you have something misconfigured in the `PATH` environment variable.

I suggest you the following: 
1. Verify that the directory of `powershell.exe` is in the `PATH` variable. Note that the directory must contain `powershell.exe` (not `pwsh.exe`)
2. If yes, check that the PATH variable does not present any syntax error.   Errors in the path can cause general system malfunction.

Please let me know if you resolve the problem.

**Reply by jacoclem** [reply]
Hi, everything seems to be working fine now, thank you very much.
Do I need to follow the steps above to exit debug mode?

Thanks again for the support
Jacopo

**Reply by Skazza94** [reply]
Hi @jacoclem,
yes, you have to go into `kathara settings` and put the logging level back to `INFO`.

Mariano.
